extensions/amp-next-page/1.0/service.js

```diff
@@ -600,7 +601,7 @@ export class NextPageService {
       const amp = this.multidocManager_.attachShadowDoc(
         shadowRoot,
         content,
-        '',
+        url,
```

Comment:
...and this PR becomes a one line change!

```suggestion
        page.url,
```

Reply:
nice!